main.go

```diff
@@ -169,7 +169,7 @@ func main() {
 	case "noop":
 		r, err = registry.NewNoopRegistry(p)
 	case "txt":
-		r, err = registry.NewTXTRegistry(p, cfg.TXTPrefix, cfg.TXTOwnerID, cfg.TXTCacheInterval)
+		r, err = registry.NewTXTRegistry(p, cfg.TXTPrefix, cfg.TXTOwnerID, cfg.TXTCacheEnabled, cfg.TXTCacheInterval)
```

Comment:
can't we just CacheInterval flag, which is `0` by default and if it is zero then we simply don't use cache. Bottom line, the less configuration, the better

Reply:
Could we use '-1' to disable ?

Reply:
@ideahitme I don't have a strong opinion on that. I just started to implement it due to the demand of having a feature flag. I'm also fine with your suggestion.

Reply:
if using the cache time to disable, could be also set that to default?